sentences/es/climate_HassClimateSetTemperature.yaml

```diff
@@ -3,7 +3,7 @@ intents:
   HassClimateSetTemperature:
     data:
       - sentences:
-        - "pon <temp> a <temperature>"
-        - "pon <temp> <area> a <temperature>"
-        - "pon <area> <temp> a <temperature>"
+        - "(pon|poner|establece|establecer|ajusta|ajustar) <temp> a <temperature>"
```

Comment:
Consider moving this to an expansion rule

Reply:
Done